byterun/Makefile.nt

```diff
@@ -59,7 +61,10 @@ libcamlrund.$(A): $(DOBJS)
 	echo " caml/freelist.h caml/minor_gc.h caml/osdeps.h caml/signals.h"\
 	  >> .depend.win32
 	cat .depend >> .depend.win32
-	sed -e '/\.d\.o/q' -e 's/^\(.*\)\.o:/\1.$$(O) \1.$$(DBGO):/' \
+	sed -ne '/\.pic\.o/q' \
```

Comment:
Using `-n` means that there isn't a stray `.pic.o` line at the bottom, hence the `-e p` at the end (or there'd be no lines at all...)